client-go/internal/jwt/jwt.go

```diff
@@ -0,0 +1,72 @@
+package jwtValidator
```

Comment:
Another violation of DRY here due to time constraints. With more time I would consider factoring this out to a shared package rather than duplicating it for both apps.